### DISCUSSION TITLE
Help text or placeholder for multiselect combobox

### DISCUSSION THREAD

**Discussion by lgalloherran** [twilio-labs/paste#2532] [Q&A] [answered]
Do we need to state with a help text or placeholder that a combobox is a 'Simple Multiselect Combobox'? 

![Captura de Pantalla 2022-07-13 a la(s) 3 17 22 p m](https://user-images.githubusercontent.com/72419222/178826693-8c4400b1-d3d2-43eb-bd2b-a707cec1b427.png)

**Comment by kelseycavitt** [comment, marked as answer]
Hi Luz! 👋 

We recommend using the placeholder text to provide examples or not use it at all, so it doesn't need to say, "Multi-select combobox" as it's not broadly supported by assistive technologies and won't display in older browsers. You can see more about our recommendation on placeholder text [here](https://paste.twilio.design/components/input).
